docs/custom_detect_languages_func.rst

```diff
@@ -0,0 +1,65 @@
+============
+Custom language detection
+============
```

Comment:
I think we could have it in this order:

1. section explaining the `detect_languages_func` parameter 
2. section explaining fasttext and langdetect integrations
3. section explaining how to create your own language detection functions.

Reply:
Hi, I have made the changes